src/Xrefcheck/Core.hs

```diff
@@ -116,12 +188,10 @@ instance Monoid FileInfoDiff where
 data FileInfo = FileInfo
   { _fiReferences :: [Reference]
   , _fiAnchors    :: [Anchor]
+  , _fiPath       :: FilePath
```

Comment:
For the moment I stored also the original FilePath here to be used in the program output, but this will be revisited after the discussion.